manifests/init.pp

```diff
@@ -302,7 +316,14 @@
   Optional[String] $email_smtp_domain = $::foreman::params::email_smtp_domain,
   Enum['none', 'plain', 'login', 'cram-md5'] $email_smtp_authentication = $::foreman::params::email_smtp_authentication,
   Optional[String] $email_smtp_user_name = $::foreman::params::email_smtp_user_name,
-  Optional[String] $email_smtp_password = $::foreman::params::email_smtp_password,
+  String $email_smtp_password = $::foreman::params::email_smtp_password,
```

Comment:
I think you didn't intend to change this line.

Reply:
no, certainly not, sorry :-)